hermes-client/src/main/java/pl/allegro/tech/hermes/client/HermesMessage.java

```diff
@@ -124,6 +126,23 @@ public void consumeHeaders(BiConsumer<String, String> consumer) {
         headers.forEach(consumer);
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
why not use Objects class for both equals and dashcode?

Reply:
and why do you need eq/hashcode in this class at all? I don't see other changes in code that would require this

Reply:
I removed this code: https://github.com/allegro/hermes/pull/1077